src/test/kotlin/uk/gov/justice/digital/hmpps/prisonregister/resource/PrisonMaintenanceResourceIntTest.kt

```diff
@@ -94,6 +109,8 @@ class PrisonMaintenanceResourceIntTest : IntegrationTest() {
         .expectBody().json("updated_prison".loadJson())
 
       verify(auditService).sendAuditEvent("PRISON_REGISTER_UPDATE", Pair("MDI", UpdatePrisonDto("Updated Prison", false)))
+      verify(snsService).sendEvent(EventType.PRISON_REGISTER_UPDATE, "MDI")
+      verify(telemetryClient).trackEvent(eq("prison-register-update"), any(), isNull())
     }
   }
```

Comment:
Next PR will add a test with a test-queue subscribing to the domain events topic, ensuring the event is received by the test queue